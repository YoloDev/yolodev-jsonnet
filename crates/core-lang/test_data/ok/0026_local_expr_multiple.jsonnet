local foo = true, bar = foo ; bar
