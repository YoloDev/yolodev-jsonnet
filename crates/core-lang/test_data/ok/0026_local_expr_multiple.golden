local foo#2 = true,
      bar#3 = foo#2;
bar#3
